### PR TITLE
VRT: Add visual tests for button component

### PR DESCRIPTION
**Pull request by Donaab**
[JIRA ticket](https://reciprocitylabs.atlassian.net/browse/)

#### Description
For now, I have added a test for the button so we can see how the tests behave on the CI and I will work on other PR to collect all the stories from the storybook as @chavlji found me some way how I can collect all the stories.
#### ChangeLOG

- [ ] I have added all notable deployment/development environment (onprem!) changes to [CHANGELOG.md](https://github.com/reciprocity/zen-ui/blob/main/README.md)

#### Useful links
[Zen UI readme](https://github.com/reciprocity/zen-ui/blob/main/README.md)

[PR review guidelines](https://github.com/reciprocity/zengrc/blob/develop/doc/dev_environment_and_process/pull_request_reviews.md)

[Git commit guidelines](https://github.com/reciprocity/zengrc/blob/develop/doc/dev_environment_and_process/commit_guidelines.md)

### CODE REVIEW

**Comment by Donaab** [comment]
Tests fail because they actually found the issue. We had buttons displayed like 
Buttons : Button - default ⋅ Storybook 2021-01-14 23-29-42

and now they are displayed like 
Buttons : Button - StorySimple ⋅ Storybook 2021-01-14 23-27-29

@chavlji @mplesec What is the expected? How the buttons should look so I know if I need to update the screenshots or to wait for the fix?

**Comment by chavlji** [comment]
@Donaab it's ok for now. Those buttons currently have no labels in those two tests.
I've already added a task to fix that https://reciprocitylabs.atlassian.net/browse/PLAT-839

File: cypress/plugins/index.js [hunk 1]
```diff
@@ -9,6 +9,30 @@ const {
 /**
  * @type {Cypress.PluginConfig}
  */
+
 module.exports = (on, config) => {
   addMatchImageSnapshotPlugin(on, config);
+
+  on('before:browser:launch', (browser, launchOptions) => {
+    if (browser.name === 'chrome' && browser.isHeadless) {
+      launchOptions.args.push('--window-size=1440,900')
```
Comment (mplesec) [inline cypress/plugins/index.js:18]: 👍

File: cypress/support/commands.js [hunk 2]
```diff
@@ -8,7 +8,9 @@ function addMatchImageSnapshotCommandInMode(isInteractiveMode) {
       cy.log('Skipping snapshot');
     });
   } else {
-    addMatchImageSnapshotCommand();
+    addMatchImageSnapshotCommand({failureThreshold: 0.03,
```
Comment (mplesec) [inline cypress/support/commands.js:11]: NIT: the `failureThreshold` should be in its own line, surprised the linter doesn't complain here

File: cypress/support/commands.js [hunk 3]
```diff
@@ -8,7 +8,9 @@ function addMatchImageSnapshotCommandInMode(isInteractiveMode) {
...
+      failureThresholdType: 'percent',
+      customDiffConfig: { threshold: 0.1 },});
```
Comment (mplesec) [inline cypress/support/commands.js:13]: I would keep this at 0 tolerance, the snapshots should be a complete match

**Review by mplesec — APPROVED** [review]
Very nice 👍

**Pull request merged**